CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `ui/MarqueeDecorator` warning against nested `Marquee`'s
+
```

Comment:
This changelog should go in `packages/ui`. I can take care of that during integration.